### ISSUE TITLE
correct firmware path as upstreamed ones

### ISSUE THREAD

**Issue by vicamo** [intel/ivsc-driver#42]
In linux-firmware upstream repo [commit cfc997c](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=cfc997c664f076c6800b749ed7ef315b1e8e76fd), ivsc firmware blobs are installed into a different path than it is expected [now](https://github.com/intel/ivsc-driver/blob/73a044d9633212fac54ea96cdd882ff5ab40573e/drivers/misc/mei/hw-vsc.c#L339). Need an update to accommodate to the new path.

**Comment by wentongwu** [comment]
@vicamo IVSC driver set has been upstreamed, could you please use the upsteamed IVSC driver and IPU6 driver? Thanks

**Comment by anthonywong** [comment]
@wentongwu I'd like to confirm can mainline kernel today (6.8-rc2 as of this writing) fully replace the ivsc driver in this repo, without any loss of functionalities?
You also mentioned ipu6 driver, but as far as we know, ipu6 driver has not been completely upstreamed yet, could you elaborate on this as well?

**Comment by wentongwu** [comment]
> You also mentioned ipu6 driver, but as far as we know, ipu6 driver has not been completely upstreamed yet, could you elaborate on this as well?

I have tested the upstream-ongoing IPU6 driver with upstreamed IVSC driver, it works well. But I don't know the exact progress of IPU6 driver upstreaming, @hao-yao could you please help update current status of IPU6 driver upstreaming? Thanks

**Comment by wentongwu** [comment]
I learned from @ccgjimlai that, IPU6 driver will make changes accordingly to fit IVSC upstreamed drivers if IPU6 drivers haven't been fully merged to 6.8.

And I suggest that we don't change the path in this repo for previous release compatibility. @vicamo

**Comment by wentongwu** [comment]
> @wentongwu I'd like to confirm can mainline kernel today (6.8-rc2 as of this writing) fully replace the ivsc driver in this repo, without any loss of functionalities? 

Yes, the upstreamed IVSC driver can fully replace this repo without any loss of functionalities, but expect according changes from IPU6 github drivers. @anthonywong

**Comment by vicamo** [comment]
> I learned from @ccgjimlai that, IPU6 driver will make changes accordingly to fit IVSC upstreamed drivers if IPU6 drivers haven't been fully merged to 6.8.
> 
> And I suggest that we don't change the path in this repo for previous release compatibility. @vicamo

People are running your dkms releases at this moment. If the upstreamed version of IVSC should be used instead from now on, I guess I'll expect a warning on the ivsc-driver README as well as the migration plan, the accommodation changes to ipu6-drivers specifically, being proposed. I did not find there is one such pull request or similar code. Please suggest.

**Comment by wentongwu** [comment]
> > I learned from @ccgjimlai that, IPU6 driver will make changes accordingly to fit IVSC upstreamed drivers if IPU6 drivers haven't been fully merged to 6.8.
> > And I suggest that we don't change the path in this repo for previous release compatibility. @vicamo
> 
> People are running your dkms releases at this moment. If the upstreamed version of IVSC should be used instead from now on, I guess I'll expect a warning on the ivsc-driver README as well as the migration plan, the accommodation changes to ipu6-drivers specifically, being proposed. I did not find there is one such pull request or similar code. Please suggest.

@ccgjimlai told me Canonical would use IVSC upstream driver for ubuntu 24.04 and IPU6 owner would make the accommodation changes, I don't have much details. @vicamo
